nx migrate next giving error @nestjs/common

### Current Behavior


 >  NX   The migrate command failed.


 >  NX   Command failed: npm install @nestjs/common@^9.1.0

   npm ERR! code ETARGET
   npm ERR! notarget No matching version found for @nestjs/common@9.1.0.
   npm ERR! notarget In most cases you or one of your dependencies are requesting
   npm ERR! notarget a package version that doesn't exist.

   npm ERR! A complete log of this run can be found in:
   npm ERR!    xxxx-debug-0.log

   Run migrate with --to="package1@version1,package2@version2"
   Pass --verbose to see the stacktrace.


### Expected Behavior

migration should be successful

### GitHub Repo

_No response_

### Steps to Reproduce

1. nx migrate next

### Nx Report

```shell
NA
```


### Failure Logs

_No response_

### Additional Information

_No response_

--- a/packages/nest/migrations.json
+++ b/packages/nest/migrations.json
@@ -47,15 +47,15 @@
       "version": "16.1.0-beta.0",
       "packages": {
         "@nestjs/common": {
-          "version": "^9.1.0",
+          "version": "^9.1.1",
           "alwaysAddToPackageJson": false
         },
         "@nestjs/core": {
-          "version": "^9.1.0",
+          "version": "^9.1.1",
           "alwaysAddToPackageJson": false
         },
         "@nestjs/platform-express": {
-          "version": "^9.1.0",
+          "version": "^9.1.1",
           "alwaysAddToPackageJson": false
         },
         "@nestjs/schematics": {
@@ -67,7 +67,7 @@
           "alwaysAddToPackageJson": false
         },
         "@nestjs/testing": {
-          "version": "^9.1.0",
+          "version": "^9.1.1",
           "alwaysAddToPackageJson": false
         }
       }

--- a/packages/nest/src/generators/init/init.spec.ts
+++ b/packages/nest/src/generators/init/init.spec.ts
@@ -1,7 +1,11 @@
 import type { Tree } from '@nx/devkit';
 import * as devkit from '@nx/devkit';
 import { createTreeWithEmptyWorkspace } from '@nx/devkit/testing';
-import { nestJsVersion, nxVersion } from '../../utils/versions';
+import {
+  nestJsSchematicsVersion,
+  nestJsVersion,
+  nxVersion,
+} from '../../utils/versions';
 import { initGenerator } from './init';
 
 describe('init generator', () => {
@@ -26,7 +30,7 @@ describe('init generator', () => {
     expect(packageJson.dependencies['tslib']).toBeDefined();
     expect(packageJson.dependencies['@nx/nest']).toBeUndefined();
     expect(packageJson.devDependencies['@nestjs/schematics']).toBe(
-      nestJsVersion
+      nestJsSchematicsVersion
     );
     expect(packageJson.devDependencies['@nestjs/testing']).toBe(nestJsVersion);
     expect(packageJson.devDependencies['@nx/nest']).toBe(nxVersion);

--- a/packages/nest/src/generators/init/lib/add-dependencies.ts
+++ b/packages/nest/src/generators/init/lib/add-dependencies.ts
@@ -1,6 +1,7 @@
 import type { GeneratorCallback, Tree } from '@nx/devkit';
 import { addDependenciesToPackageJson } from '@nx/devkit';
 import {
+  nestJsSchematicsVersion,
   nestJsVersion,
   nxVersion,
   reflectMetadataVersion,
@@ -20,7 +21,7 @@ export function addDependencies(tree: Tree): GeneratorCallback {
       tslib: tsLibVersion,
     },
     {
-      '@nestjs/schematics': nestJsVersion,
+      '@nestjs/schematics': nestJsSchematicsVersion,
       '@nestjs/testing': nestJsVersion,
       '@nx/nest': nxVersion,
     }

--- a/packages/nest/src/utils/versions.ts
+++ b/packages/nest/src/utils/versions.ts
@@ -1,6 +1,7 @@
 export const nxVersion = require('../../package.json').version;
 
-export const nestJsVersion = '^9.1.0';
+export const nestJsVersion = '^9.1.1';
+export const nestJsSchematicsVersion = '^9.1.0';
 export const rxjsVersion = '^7.8.0';
 export const reflectMetadataVersion = '^0.1.13';
 export const tsLibVersion = '^2.3.0';

